pkg/db/src/__tests__/tests.ts

```diff
@@ -1132,8 +1134,10 @@ export const booksInterface: TestFunction = async (db) => {
 		price: 39.86
 	};
 
-	const booksInterface = db.books();
+	const pluginManagerInstance = new PluginManager();
+	const booksInterface = db.books(pluginManagerInstance);
 
+	pluginManagerInstance.disable();
```

Comment:
This is totally unnecessary in tests and is something I've suggested as a (click through) debug measure. In tests, you can simply set the transactions into the note, without fetching book data.